dts/bindings/gpio/nordic,nrf-gpio.yaml

```diff
@@ -5,7 +5,7 @@
 #
 ---
 title: NRF5 GPIO
```

Comment:
NRF5 -> nRF
Same applies to [line 12](https://github.com/zephyrproject-rtos/zephyr/pull/8729/files#diff-75b66b32098c53e8beb8efb05cc97588R12).